### DISCUSSION TITLE
Unable to register and log in

### DISCUSSION THREAD

**Discussion by khanh-duy-003** [vercel/community#7341] [Account Issue] [answered]
### Question

when I signed up and received the message: Your account requires further verification. Please contact registration@vercel.com to complete your registration. help me please

**Comment by pawlean** [comment, marked as answer]
Hi @khanh-duy-003!

Have you contacted us via [registration@vercel.com](mailto:registration@vercel.com)? If so, our team will get back to you shortly.

**Reply by khanh-duy-003** [reply]
Contacted but received a message that the email does not exist. Please contact me for help

**Reply by pawlean** [reply]
I flagged this internally, and found there is an issue from our side. We're working on fixing this now. Thanks for your patience!

Update: false alarm! It looks like it may be an issue with your email. Can you email us from another address? @khanh-duy-003

**Comment by khanh-duy-003** [comment]
Hi!
Contacted but received a message that the email does not exist. Please
contact me for help

Vào Th 2, 1 thg 7, 2024 vào lúc 23:12 Pauline P. Narvas <
***@***.***> đã viết:

**Comment by CCTSCompany** [comment]
Hi @pawlean! How long does it usually take to hear back from registration@vercel.com? I sent them an email on 6/28 and still haven't got any response. Got the same issue ("couldn't complete sign-up" etc). No messages about non-existing email though.

**Reply by CCTSCompany** [reply]
Just got a response, all good!

**Reply by pawlean** [reply]
Thanks @CCTSCompany!

[a comment by an automated account is not shown]
